src/services/JobRoleService.ts

```diff
@@ -17,4 +18,14 @@ export const getAllOpenJobRoles = async (): Promise<JobRoleResponse[]> => {
             throw new Error(e.message);
         }
     }
+}
+
+export const getJobRoleById = async (id: string): Promise <JobRoleDetailResponse> => {
+    try {
+        const response: AxiosResponse = await axios.get(URL + "/" + id);
+        return response.data;
+
+    } catch (e) {
+        throw new Error("Could not get job role");
```

Comment:
is this user friendly?

Reply:
fixed

Reply:
changed it to: We encountered a problem while trying to retrieve this job role. Please try again later!